packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -270,11 +270,13 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
       );
     }
 
+    // Remove the leading [BulletSpacer] if in narrow mode.
+    final actions = isNarrow ? widget.actions.sublist(1) : widget.actions;
```

Comment:
Perhaps switch this code to only adding the BulletSpacer() if we're not in narrow mode? That way this code won't need to make assumptions about the first item in the actions list.

Reply:
We can't calculate isNarrow in app.dart (where we create the tabs list) because MediaQuery.of(context) would be called with a context that does not contain a media query.

Reply:
ok, gotcha.

Reply:
But couldn't we remove the code that added the BulletSpacer() to the original actions list, and here, do something like:

var actions = widget.actions;

```
if (!isNarrow) {
  actions = [BulletSpacer()]..addAll(actions);
}
```

Reply:
done